dashboard/src/features/accessControl/actions.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { chainClient } from 'utility/environment'
+import { baseListActions } from 'features/shared/actions'
+import { actions as appActions } from 'features/app'
+import { push } from 'react-router-redux'
+import TokenCreateModal from './components/TokenCreateModal'
+
+const baseActions = baseListActions('accessControl', {
+  clientApi: () => chainClient().accessControl
+})
+
+export default {
+  fetchItems: () => {
+    return (dispatch) => {
+      const promise = chainClient().accessControl.list()
```

Comment:
Good catch. This is repurposed from the old `baseListActions` version, which was more complex.